package.json

```diff
@@ -16,35 +16,40 @@
   "jest": {
     "testURL": "http://127.0.0.1:3000/",
     "transform": {
-      ".(ts|tsx)": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+      ".(ts|tsx)$": "ts-jest"
     },
     "testPathIgnorePatterns": [
       "/node_modules/",
       "/lib/",
       "/test/integration/"
     ],
-    "testRegex": "(/test/.*|\\.(test|spec))\\.(ts|tsx|js)$",
+    "testRegex": "(/test/.*(spec).*|\\.(test|spec))\\.(ts|tsx|js)$",
     "moduleFileExtensions": [
       "ts",
       "tsx",
       "js",
       "jsx",
-      "json"
+      "json",
+      "node"
     ],
     "moduleNameMapper": {
-      "\\.(scss|css|less)$": "identity-obj-proxy",
+      "\\.(scss|css|less|svg|png|jpg)$": "identity-obj-proxy",
```

Comment:
what is this for?

Reply:
this is for jest to understand what to do with svg  png jpg files (namely: nothing at all)